Entities/Actor.h

```diff
@@ -914,16 +914,12 @@ ClassInfoGetters
 
 	const std::deque<MovableObject *> * GetInventory() { return &m_Inventory; }
 
+	/// <summary>
+	/// Returns the maximum total mass this Actor can carry in it's inventory.
+	/// </summary>
+	/// <returns> The maximum carriable mass of this Actor.</returns>
```

Comment:
Should be `its` instead of `it's` in the summary
Also, remove the space at the start of the returns comment